utils/html.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"bytes"
+	"strings"
+	"unicode"
+
+	"golang.org/x/net/html"
+)
+
+// This is non-exhaustive, but captures most required nodes
+const stripAllowedNodes = "html body p a strong em span section div"
+
+// StripHTML strips HTML tags from a string, extracting all plain text.
+func StripHTML(htmls string) (string, error) {
```

Comment:
This is horrendous and mostly exists to stop the horrors of the HTML-based show description from screwing up tooltips.